test/unittests/test_xmlupload.py

```diff
@@ -148,8 +148,7 @@ def test_remove_circular_references(self) -> None:
         stashed_xml_texts_expected = {
             "test_thing_1": {
                 "testonto:hasRichtext": [
-                    "\n                This is <em>bold and <strong>strong</strong></em> text! "
-                    "It contains links to all resources:\n"
+                    "\n                This text contains links to all resources:\n"
```

Comment:
Here, I separated 2 things: 

- "test_xmlupload.py" should use a text value that contains only some links, because that's the thing to be tested in this test.
- the `<em>` tags etc. are now moved into another property (see below), so that if something changes there, this test here doesn't have to be adapted.